benches/benches/vm_initialization.rs

```diff
@@ -75,12 +96,13 @@ fn transaction<R: Rng>(
 
 pub fn vm_initialization(c: &mut Criterion) {
     let mut rng = StdRng::seed_from_u64(8586);
-    let consensus_params = ConsensusParameters::default();
+    let mut consensus_params = ConsensusParameters::default();
+    consensus_params.set_tx_params(TxParameters::default().with_max_size(256 * 1024));
```

Comment:
Where does `256 * 1024` come from?

Reply:
From the current configuration of the network. Testnet currently allows 256KB